RecoHI/HiTracking/python/hipixeltracks_miniAOD_cfi.py

```diff
@@ -0,0 +1,17 @@
+import FWCore.ParameterSet.Config as cms
+
+hipixeltracks = cms.EDProducer("HIPixelTracksSlimmer_miniAOD",
```

Comment:
the explicit construction should be replaced with a clone from the module produced by the fillDescriptions method,
with appropriate parameter value replacement if any

Reply:
Okay, this is fixed now in this commit: https://github.com/cms-sw/cmssw/pull/30313/commits/deebf6766c730a8c97ac3f3fe90b065b24d5e70d